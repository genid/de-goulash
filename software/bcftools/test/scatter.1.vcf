##fileformat=VCFv4.2
##reference=ref.fasta
##contig=<ID=21,length=48129895>
##contig=<ID=22,length=51304566>
##contig=<ID=X,length=155270560>
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	A	B
21	10	.	C	A	.	.	.	GT	0/0	1/1
21	22	.	A	C	.	.	.	GT	1/1	0/0
21	23	.	C	A	.	.	.	GT	0/0	0/0
21	24	.	A	C	.	.	.	GT	1/1	1/1
22	10	.	C	A	.	.	.	GT	0/1	0/0
22	22	.	A	C	.	.	.	GT	0/0	0/1
22	23	.	C	A	.	.	.	GT	0/0	1/0
X	10	.	A	C	.	.	.	GT	0/1	0/1
X	22	.	C	A	.	.	.	GT	0/1	0/1
