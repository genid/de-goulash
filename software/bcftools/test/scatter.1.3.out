#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	A	B
21	10	.	C	A	.	.	.	GT	0/0	1/1
21	22	.	A	C	.	.	.	GT	1/1	0/0
21	23	.	C	A	.	.	.	GT	0/0	0/0
21	24	.	A	C	.	.	.	GT	1/1	1/1
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	A	B
22	10	.	C	A	.	.	.	GT	0/1	0/0
22	22	.	A	C	.	.	.	GT	0/0	0/1
22	23	.	C	A	.	.	.	GT	0/0	1/0
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	A	B
X	10	.	A	C	.	.	.	GT	0/1	0/1
X	22	.	C	A	.	.	.	GT	0/1	0/1
